deployments/gcp/dc-only/terraform.tfvars.sample

```diff
@@ -43,6 +50,9 @@ gcp_service_account  = "service_account_name@<project_id>.iam.gserviceaccount.co
 #    3. unicode characters
 # See: https://docs.microsoft.com/en-us/windows/security/threat-protection/security-policy-settings/password-must-meet-complexity-requirements
 
+# If using encryption, enter the KMS key ID. Otherwise, comment out.
+# kms_cryptokey_id = "projects/<project-id>/locations/<location>/keyRings/<keyring-name>/cryptoKeys/<key-name>"
```

Comment:
Same comment as above. "If using encryption, enter the KMS key ID and uncomment the line" would be more specific.